Assets/AppCenter/Plugins/iOS/Distribute/DistributeDelegate.h

```diff
@@ -7,11 +7,14 @@
 #import <AppCenterDistribute/AppCenterDistribute.h>
 
 typedef bool (__cdecl *ReleaseAvailableFunction)(MSACReleaseDetails*);
+typedef void (__cdecl *NoReleaseAvailableFunction)();
 
 @interface UnityDistributeDelegate : NSObject<MSACDistributeDelegate>
 
 - (BOOL)distribute:(MSACDistribute *)distribute releaseAvailableWithDetails:(MSACReleaseDetails *)details;
+- (void)noReleaseAvailable;
```

Comment:
This API was renamed to 
`(void)distributeNoReleaseAvailable:(MSACDistribute *)distribute; ` ([PR](https://github.com/microsoft/appcenter-sdk-apple/pull/2238/files#diff-dc19369dffd7a467cc08789d92a9d8217f62b599d3b02d7bf67a8a9497dab0a4R36))